android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/shopping/ProductRecommendation.kt

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.concept.engine.shopping
+
+/**
+ * Contains information about a product recommendation.
+ */
+interface ProductRecommendation {
+
+    /**
+     * Url of recommended product.
+     */
+    val url : String
```

Comment:
Is this also missing the rest of the fields?

Reply:
It's also the concept interface, similar to the previous patch